jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/viewmodel/ChapterPageViewModel.kt

```diff
@@ -428,6 +430,15 @@ class ChapterPageViewModel : ViewModel() {
         }
     }
 
+    private fun setMarker(marker: String, take: Take) {
+        AudioFile(take.file).apply {
+            if (metadata.getCues().isEmpty()) {
+                metadata.addCue(0, marker)
+                update()
+            }
+        }
+    }
+
```

Comment:
adapt with existing takes which have not previously contained marker in metadata